<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatriceIP</class>
 <widget class="QWidget" name="CalculatriceIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatriceIP</string>
  </property>
  <widget class="QPushButton" name="pushButtonCalculer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse IPV4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPV4_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPV4_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPV4_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPV4_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Suffixe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Masque</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre maximum d'adresses IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse réseau</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse IP utilisables</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre maximum de sous réseaux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse de diffusion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AddressReseau">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AddressDiffusion">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IPutilisable">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Masque">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SousReseau">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_AdresseIp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Suffixe">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>72</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
